src/base/bittorrent/trackerentry.cpp

```diff
@@ -52,6 +53,30 @@ QString TrackerEntry::url() const
     return QString::fromStdString(nativeEntry().url);
 }
 
+QStringList TrackerEntry::messages() const
+{
+    QStringList messages;
+    const auto addMessage = [&messages](const std::string &msg) {
+        QString message = QString::fromStdString(msg).trimmed();
+        if (!message.isEmpty() && !messages.contains(message))
+            messages << std::move(message);
+    };
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    addMessage(nativeEntry().message);
+    if (messages.isEmpty() && status() == TrackerEntry::NotWorking)
```

Comment:
please add parentheses to the comparison. And to similar code below too.

Reply:
Done